metacatalog/api/find.py

```diff
@@ -133,6 +137,8 @@ def find_keyword(session: 'Session', id: Optional[int] = None, uuid: Optional[st
     # add needed filter
     if id is not None:
         query = query.filter(models.Keyword.id==id)
+    if full_path is not None:
+        query = query.filter(models.Keyword.full_path==full_path)
```

Comment:
This way, the path has to be a perfect match. Is that a good strategy? For other text fields, we implemented a logic that switches to a `LIKE` statement if the filter statement included a ' *' or '%'. 
Doesn't that make sense here as well?